docs/user_guide/command_line_interface.rst

```diff
@@ -289,18 +291,25 @@ The available options:
 \-\-data_directory <data_directory>
 +++++++++++++++++++++++++++++++++++
 ``--data_directory`` is an **optional** argument. It specifies the directory where ``marius_preprocess`` stores
-preprocessed data.
+preprocessed data. 
 
 \-\-dataset <dataset>, \-d <dataset>
 ++++++++++++++++++++++++++++++++++++
-``--dataset`` is an **optional** argument. It specifies the name of the supported dataset. It should not be
-used when ``--stats`` is in use.
+``--dataset`` is an **optional** argument. This argument is used when users want to 
+generate a Marius configuration file for a built-in dataset.
+It specifies the name of the built-in dataset which its configuration file will be generated. 
```

Comment:
Nit:

"It specifies the name of the built-in dataset for which the configuration file will be generated."